### ISSUE TITLE
Agenda visibility

### ISSUE THREAD

**Issue by f0x2501** [eSkiSo/Subtaskdate#3]
Can you plan to add subtasks with due date appear in the agenda ?
It'll be very appreciate ;)

Thanks for your great software !

**Comment by creecros** [comment]
@mentat-fr 

Not sure I understand what this request means. What is the agenda?

**Comment by f0x2501** [comment]
Hi, it was to integrate due date in the KB calendar and ical export.
It'll be great to have subtasks due date mixe with project due date.
Hope is more clear.

**Comment by creecros** [comment]
@mentat-fr 

https://github.com/eSkiSo/Subtaskdate/releases/tag/1.1.0

Added Calendar Events. I have no idea how to use iCal, so I have no idea if it works with that as well. But it does work for the embeded calendar for kanboard versions less than 1.2.4, and with the Plugin-Calendar for Newer Kanboard Versions after the calendar became a plugin.

Let me know.

**Comment by creecros** [comment]
Oh and to make noticable, the subtask will appear only on due date of subtask with Grey background/Red Border, titled the name of the Subtask, and link url will direct to task.

![image](https://user-images.githubusercontent.com/26339368/49603348-0240b200-f959-11e8-9ff6-041c900fdcf4.png)

**Comment by creecros** [comment]
@mentat-fr 

Ok, looked a bit deeper, the ICalendarController controls the events for ICal, and it has no merge hooks like the CalendarController. I'll have to bypass that controller and do some studying, to figure out what the array should look like to send to the formatter.

**TLDR;**

Today, celebrate that Events were added to the Calendar :tada:, and one day I'll get to the ICal events.

**Comment by f0x2501** [comment]
Sorry for this to late reply, big thanks and kudo to you ;)
